index.html

```diff
@@ -0,0 +1,322 @@
+          </div>
+          <div class="item-heading">Customization</div>
+          <div class="item-description">
+            We deliver outsourced aviation services for military customers
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- Destinations -->
+    <section id="destination">
+      <div class="title">Top Selling</div>
+      <div class="subtitle">Top Destinations</div>
+      <div class="college-container">
+        <div class="dest-college">
+          <div class="image">
+            <img src="./images/rome.png" alt="Rome" />
+          </div>
+          <div class="college-item-1">
+            <div class="name">Rome, Italy</div>
+            <div class="price">$5,42k</div>
+          </div>
+          <div class="college-item-2">
+            <i class="fa-solid fa-location-arrow"></i>
+            <span>10 days trip</span>
+          </div>
+        </div>
+        <div class="dest-college">
+          <div class="image">
+            <img src="./images/london.jpg" alt="london" />
+          </div>
+          <div class="college-item-1">
+            <div class="name">London,UK</div>
+            <div class="price">$4.2k</div>
+          </div>
+          <div class="college-item-2">
+            <i class="fa-solid fa-location-arrow"></i>
+            <span>12 days trip</span>
+          </div>
+        </div>
+        <div class="dest-college">
+          <div class="image">
+            <img src="./images/europe.png" alt="europe" />
+          </div>
+          <div class="college-item-1">
+            <div class="name">Full Europe</div>
+            <div class="price">$15k</div>
+          </div>
+          <div class="college-item-2">
+            <i class="fa-solid fa-location-arrow"></i>
+            <span>28 days trip</span>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- book trip -->
+    <section id="tripsteps">
+      <div class="title">Easy and Fast</div>
+      <div class="container-steps">
+        <div class="left-item">
+          <div class="title">Book your next trip in 3 Easy Steps</div>
+          <div class="item-container">
+            <div class="icon">
+              <img src="./images/Rectangle 21.png" alt="destination" />
+            </div>
+            <div class="description">
+              <div class="heading">Choose Destination</div>
+              <div class="head-description">
+                Lorem ipsum dolor sit amet, consectetur <br/> adipisicing elit. Atque,
+                magnam!
+              </div>
+            </div>
+          </div>
+          <div class="item-container">
+            <div class="icon">
+              <img src="./images/Group 12.png" alt="" />
+            </div>
+            <div class="description">
+              <div class="heading">Make Payment</div>
+              <div class="head-description">
+                Lorem ipsum dolor sit amet, consectetur <br/> adipisicing elit. Atque,
```

Comment:
Replace tis random text with actual content

Reply:
i've written same text given in the image
![Home Page](https://github.com/ajay-keelu/Tez_Task_Frontend/assets/84334673/6b1a8e18-a321-461e-97d5-ac23fff2bd17)